.github/workflows/release.yml

```diff
@@ -105,15 +101,12 @@ jobs:
           cp target/${{ matrix.target }}/release/git-remote-rad "$staging"/
           tar czf "$staging.tar.gz" "$staging"
           cp "$staging.tar.gz" "$staging"/
-      - id: 'auth'
-        uses: 'google-github-actions/auth@v1'
-        with:
-          workload_identity_provider: 'projects/281042598092/locations/global/workloadIdentityPools/github-actions/providers/google-cloud'
-          service_account: 'github-actions@radicle-services.iam.gserviceaccount.com'
-      - name: 'Set up Cloud SDK'
-        uses: 'google-github-actions/setup-gcloud@v1'
-      - id: 'upload-file'
-        uses: 'google-github-actions/upload-cloud-storage@v1'
-        with:
-          path: ${{ matrix.target }}
-          destination: heartwood-artifacts/${{ github.sha }}
+      - shell: bash
+        run: |
+          echo "$SSH_PRIVATE_KEY" >ssh-private-key
+          chmod go-rwx ssh-private-key
+          ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i ssh-private-key github-actions@files.radicle.xyz mkdir -p "/mnt/HC_Volume_30083854/releases/${{ matrix.target }}/${{ github.sha }}"
```

Comment:
What is this `HC_Volume` thing, it seems like something that would break if we changed the volume?

Reply:
It’s a Hetzner mounted volume.  Those volumes are supposed to outlive VMs so I doubt it’s ever going to change